invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme.less

```diff
@@ -418,3 +416,32 @@ body {
     }
   }
 }
+
+.record-citation {
```

Comment:
since it is only one in the page, maybe we could use an `id` to group classes for the same style. This is also faster as CSS selector than CSS classes `.`. e.g:

```
#record-citation: {
...

    .citation-style-selector {...}

}